doc/develop/west/build-flash-debug.rst

```diff
@@ -476,6 +562,23 @@ For example, to print usage information about the ``jlink`` runner::
 
   west flash -H -r jlink
 
+.. _west-multi-domain-flashing:
+
+multi-domain flashing
```

Comment:
Please fix case and resolve inconsistent terminology

Reply:
tried to fix inconsistency, please check if you still find inconsistency in this section.